app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -273,6 +277,19 @@ def process_api_features_support
       end
     end
 
+    def raw_connect(options)
+      version = options[:version]
```

Comment:
How will the client of this method decide which version to pass here?
We use this https://github.com/jntullo/manageiq-providers-ovirt/blob/3930da25d85fd020cbcc29ecb5a72b142ee933c3/app/models/manageiq/providers/redhat/infra_manager/api_integration.rb#L19
in the connect method.